src/tasks/wifi.c

```diff
@@ -52,6 +55,8 @@
 #define STACK_SIZE		256
 /* Make all task names 16 chars including NULL char */
 #define THREAD_NAME		"WiFi Task      "
+/* The highest channel WiFi can use (USA) */
+#define WIFI_MAX_CHANNEL	11
 
```

Comment:
Yep.  It also depends on what the esp8266 supports, which currently is only 1 - 11 AFAICT.